prog/vnet/update_firewall_rules.rb

```diff
@@ -86,17 +86,26 @@ class Prog::Vnet::UpdateFirewallRules < Prog::Base
     ip daddr @globally_blocked_ipv4s drop
     ip6 daddr @globally_blocked_ipv6s drop
     ip saddr @private_ipv4_cidrs ct state established,related,new counter accept
-    ip daddr @private_ipv4_cidrs ct state established,related counter accept
+
+    # If we are using clover_ephemeral, that means we are using ipsec. We need
+    # to allow traffic for the private communication and block via firewall
+    # rules through @allowed_ipv4_port_tuple and @allowed_ipv6_port_tuple in the
+    # next section of rules.
+    ip6 daddr #{clover_ephemeral} ct state established,related,new counter accept
+    ip6 saddr #{clover_ephemeral} ct state established,related,new counter accept
```

Comment:
can we simplify? IPsec isn't a connection tracked thing, it's not TCP. I'm actually surprised this does anything?